### PR TITLE
Clear Browsing Data

### PR DESCRIPTION
**Pull request by boek**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

**Comment by boek** [comment]
Will follow up with cleaner delete code for collections when this lands --> https://github.com/mozilla-mobile/android-components/issues/3118

File: app/src/main/java/org/mozilla/fenix/settings/DeleteBrowsingDataFragment.kt [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+        delete_browsing_data_wrapper.isClickable = true
+        delete_browsing_data_wrapper.alpha = ENABLED_ALPHA
+
+        updateTabCount()
+        updateHistoryCount()
+        updateCollectionsCount()
+    }
+
+    private fun updateDeleteButton() {
+        val openTabs = view!!.open_tabs_item!!.isChecked
```
Comment (sblatz) [inline app/src/main/java/org/mozilla/fenix/settings/DeleteBrowsingDataFragment.kt]: nit: rename these to be `openTabsChecked`, etc.

File: app/src/main/res/drawable/ic_delete.xml [hunk 2]
```diff
@@ -1,14 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekager) [inline app/src/main/res/drawable/ic_delete.xml]: plz no remove license

File: app/src/main/res/drawable/ic_delete.xml [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
-      android:fillColor="?destructive" />
+      android:strokeWidth="1"
+      android:fillColor="#20123A"
+      android:fillType="evenOdd"
```
Comment (ekager) [inline app/src/main/res/drawable/ic_delete.xml]: can we remove this evenOdd and put back the destructive fill color here? Confused why this is updated

Comment (boek) [reply app/src/main/res/drawable/ic_delete.xml]: also very confused how this got updated :|

Comment (boek) [reply app/src/main/res/drawable/ic_delete.xml]: Ooo overwrote
oops.

**Pull request merged**